addons/project/models/project_task_type.py

```diff
@@ -109,38 +172,12 @@ def _compute_disabled_rating_warning(self):
             else:
                 stage.disabled_rating_warning = False
 
+    @api.depends('project_ids')
+    def _compute_user_id(self):
+        # If project_ids is set after stage creation (e.g. when setting demo data), the default user_id has to be removed
```

Comment:
Could you please elaborate the comment a little bit ? At first sight I thought you were erasing the project manager for all the projects sharing the same stage. :)